autopush/settings.py

```diff
@@ -204,18 +206,35 @@ def message(self, value):
         """Setter to set the current message table"""
         self.message_tables[self.current_msg_month] = value
 
+    def add_tomorrow(self, today, this_month_table):
+        """check to see if tomorrow is a new month, in case we're a node that
+        has a slow clock."""
+        tomorrow = get_rotating_message_table(
+            self._message_prefix, 0, today + datetime.timedelta(days=1))
```

Comment:
We could avoid adding code to the get_rotating_message_table by just doing this check here.

```python
if (today + datetime.timedelta(days+1).month != today.month:
    next_month = get_rotating_message_table(delta=1)
    self.message_tables[next_month.table_name] = Message(next_month, self.metrics)
```

This also won't require passing this months table.

Why is it better to increase the complexity of the get_rotating_message_table function instead?